snippets/sdk/python/deployments/with_logging.py

```diff
@@ -0,0 +1,127 @@
+#! /usr/local/env python3
+
+import os
+
+from azure.identity import DefaultAzureCredential
+from azure.mgmt.authorization import AuthorizationManagementClient
+from azure.mgmt.nginx import NginxManagementClient
+from azure.mgmt.storage import StorageManagementClient
+
+import prerequisites
+
+
+def main():
+    SUBSCRIPTION_ID = os.environ.get("AZURE_SUBSCRIPTION_ID", None)
+    GROUP_NAME = "myResourceGroup"
+    DEPLOYMENT_NAME = "myDeployment"
+    STORAGE_ACCOUNT_NAME = "mystorageaccountxyzaaa"
+    BLOB_CONTAINER_NAME = "mylogcontainer"
+
+    # Create clients
+    # For other authentication approaches, please see: https://pypi.org/project/azure-identity/
+    auth_client = AuthorizationManagementClient(
+        credential=DefaultAzureCredential(), subscription_id=SUBSCRIPTION_ID
+    )
+    nginx_client = NginxManagementClient(
+        credential=DefaultAzureCredential(), subscription_id=SUBSCRIPTION_ID
+    )
+    storage_client = StorageManagementClient(
+        credential=DefaultAzureCredential(), subscription_id=SUBSCRIPTION_ID
+    )
+
+    # Create prerequisite resources
+    subnet, public_ip, identity = prerequisites.create()
+
+    # Create storage account
+    # https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples/storage
+    storage_client.storage_accounts.begin_create(
+        GROUP_NAME,
+        STORAGE_ACCOUNT_NAME,
+        {
+            "sku": {"name": "Standard_GRS"},
```

Comment:
try not to recommend this SKU. GRS might cost the user more. Leverage the simplest/default LRS.